styles/mixins/mixins.scss

```diff
@@ -0,0 +1,52 @@
+@import '../colors/colors.scss';
+
+// Mixin for centering content horizontally and vertically
+@mixin center-content {
```

Comment:
good to see, this was created as mixin,

think on more generalised way of creating with parameters